packages/cfd/src/features/Containers/trade-modal.tsx

```diff
@@ -0,0 +1,343 @@
+import React from 'react';
+import { Text, Icon, Money } from '@deriv/components';
+import { TTradingPlatformAccounts, TCFDDashboardContainer, TCFDsPlatformType } from 'Components/props.types';
+import { DetailsOfEachMT5Loginid } from '@deriv/api-types';
+import { getCFDAccountKey, isMobile, mobileOSDetect } from '@deriv/shared';
+import { localize, Localize } from '@deriv/translations';
+import { getPlatformQRCode, PlatformsDesktopDownload, mobileDownloadLink } from '../../Helpers/config';
+import { getTitle, platformsText, CTRADER_DESKTOP_DOWNLOAD } from '../../Helpers/constants';
+import { CFD_PLATFORMS, MOBILE_PLATFORMS } from '../../Helpers/cfd-config';
+import SpecBox from '../../Components/specbox';
+import PasswordBox from '../../Components/passwordbox';
+import { TCFDPasswordReset } from '../../Containers/props.types';
+
+type TTradeModalProps = {
+    mt5_trade_account: Required<DetailsOfEachMT5Loginid>;
+    is_eu_user: boolean;
+    onPasswordManager: (
+        arg1: string | undefined,
+        arg2: string,
+        group: TCFDPasswordReset['account_group'],
+        arg4: string,
+        arg5: string | undefined
+    ) => void;
+    toggleModal: () => void;
+    dxtrade_tokens: TCFDDashboardContainer['dxtrade_tokens'];
+    ctrader_tokens: TCFDDashboardContainer['ctrader_tokens'];
+    derivez_tokens: TCFDDashboardContainer['derivez_tokens'];
```

Comment:
there is derivez code here 😅 is this getting removed with @mahdiyeh-deriv 's derivez removal pr

Reply:
This is just the Trade-modal file added in the features folder. As the list should be transformed in the hooks?
Yes it will be removed in the derivez PR.